tests/resources/example_project/conda.yaml

```diff
@@ -4,3 +4,4 @@ channels:
 dependencies:
   - pip:
     - mlflow
+  - python=3.6
```

Comment:
Fix the Python version to avoid differences in behavior for users with conda2/conda3, mirroring changes we made in https://github.com/databricks/mlflow/pull/96